plugin/build.gradle.kts

```diff
@@ -117,3 +127,10 @@ tasks.publishPlugins {
 tasks.publish {
     dependsOn(tasks.publishPlugins)
 }
+
+tasks.check {
+    dependsOn(
+        // TODO: https://youtrack.jetbrains.com/issue/KT-78525
```

Comment:
```suggestion
```
I'm fine with declaring this explicitly, don't need to keep a track to that issue ;)